solutions/YS Vertex Set Path Composite.mdx

```diff
@@ -0,0 +1,251 @@
+---
+id: ys-VertexSetPathComposite
```

Comment:
rename this file so that the name is the same as the id; in this case, ys-VertexSetPathComposite.mdx
(this is a new file naming convention)